ckcp/hack/util/reset_ckcp.sh

```diff
@@ -0,0 +1,288 @@
+      done
+    fi
+    openshift_pipelines_operator=$(kubectl get operator | grep -ie "openshift-pipelines-operator" | cut -d " " -f 1)
+    if [[ -n "$openshift_pipelines_operator" ]];
+    then
+      kubectl delete operator "$openshift_pipelines_operator"
+    fi
+
+    printf "\n  Uninstalling cert-manager Operator:\n"
+    kubectl delete -k "$PLNSVC_DIR/operators/cert-manager"
+    mapfile -t cert_manager_crds < <(kubectl get crd | grep -iE "cert-manager.io|certmanagers" | cut -d " " -f 1)
+    if [[ "${#cert_manager_crds[@]}" -gt 0 ]];
+    then
+      for crd in "${cert_manager_crds[@]}"
+      do
+        kubectl delete crd "$crd" &
+        kubectl patch crd "$crd" --type json --patch='[ { "op": "remove", "path": "/metadata/finalizers" } ]' >/dev/null 2>&1
+      done
+    fi
+    cert_manager_operator=$(kubectl get operator | grep -ie "cert-manager-operator" | cut -d " " -f 1)
+    if [[ -n "$cert_manager_operator" ]];
+    then
+      kubectl delete operator "$cert_manager_operator"
+    fi
+
+    # Checks if the operators are uninstalled successfully
+    mapfile -t operators < <(kubectl get operators | grep -iE "cert-manager-operator|openshift-gitops-operator|openshift-pipelines-operator" | cut -d " " -f 1)
+    if (( ${#operators[@]} >= 1 )); then
+        printf "\n\n[ERROR] Couldn't uninstall all Operators, please try removing them manually."
+        exit
+    fi
+    printf "\nAll the operators are successfully uninstalled.\n"
+}
+
+# Cleans ckcp deployed resources from the cluster
+uninstall_ckcp(){
+    # Extract label from synctargets inside the kcp user workspace to be able to select namespaces
+    # created in the compute cluster by the same kcp-workspace.
+    ns_label=$(KUBECONFIG=${KCP_KUBECONFIG} kubectl get synctargets.workload.kcp.dev -o yaml | yq '.items[0].metadata.labels' | grep -ie "internal.workload.kcp.dev/key")
+    if [[ -z $ns_label ]]; then
+        printf "\n\n[ERROR] Couldn't extract labels from kcp synctarget object.\n"
+        printf "Make sure you pass the correct kcp_kubeconfig directory using --kcp_kubeconfig arg"
+        exit
+    fi
+
+    # remove resources created by ckcp
+    printf "\n  Uninstalling ckcp from the cluster:\n"
+    kubectl delete -k "$CKCP_DIR/openshift/overlays/dev"
+    # Check if the ckcp pod is removed from the compute cluster
+    mapfile -t ckcp_pod < <(kubectl get pods -n ckcp -l "app=kcp-in-a-pod" | cut -d " " -f 1 | tail -n +2)
+    if (( ${#ckcp_pod[@]} >= 1  )); then
+        printf "\n\n[ERROR] Couldn't remove the ckcp pods: %s ${ckcp_pod[*]}"
+        exit
+    fi
+
+    # remove syncer resources
+    printf "\n  Removing resources taken up by the syncer:\n"
+    current_context="$(yq e ".current-context" <"$KUBECONFIG")"
+    current_context=$(echo "$current_context" | sed  's,default/,,g; s,:6443/kube:admin,,g')
+    kubectl delete -f /tmp/syncer-"$current_context".yaml
```

Comment:
Where does this file comes from? What happens if I reboot my laptop and /tmp is cleaned? I would avoid on using resources that are not in the work dir or not in the repo.